phpstan.neon.dist

```diff
@@ -0,0 +1,18 @@
+parameters:
+    level: 7
+    paths:
+        - %currentWorkingDirectory%/DependencyInjection
+        - %currentWorkingDirectory%/Tests
+
+    ignoreErrors:
+        - '~.*TreeBuilder.*getRootNode.*~'
+        - '~.*TreeBuilder::root.*~'
+        - '~Parameter \#1 \$configs.*DoctrineMigrationsExtension::load.*~'
+
+    excludes_analyse:
+        - Tests/Fixtures/CustomEntityManager.php
+
+includes:
+    - vendor/phpstan/phpstan-strict-rules/rules.neon
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+    - vendor/phpstan/phpstan-phpunit/rules.neon
```

Comment:
Is the `.dist` extension really useful? Why would people need to be able to not commit changes here?

Reply:
It is a common practice to have dist file it seems.... i'm not an expert of phpstan, but this is the same as it is in the doctrine/migrations library

Reply:
ok, let's leave it as is…